hathor/transaction/storage/transaction_storage.py

```diff
@@ -898,6 +937,16 @@ def get_value(self, key: str) -> Optional[str]:
         """
         raise NotImplementedError
 
+    def get_network(self) -> Optional[str]:
+        """ Save full verification on storage
```

Comment:
Get network from storage?

Reply:
Oh! I copied the method and forgot to write the docstring.

Reply:
Done.